src/sentry/static/sentry/app/views/usageStats/types.tsx

```diff
@@ -1,7 +1,27 @@
 import {MinimalProject} from 'app/types';
 
+export enum Outcome {
+  ACCEPTED = 'accepted',
+  FILTERED = 'filtered',
+  INVALID = 'invalid',
+  DROPPED = 'dropped',
+}
+
+/**
+ * Raw response from API endpoint
+ */
+export type UsageSeries = {
```

Comment:
i'm a typescript newb but there's `SessionApiResponse` in types/index.tsx that we could generalize to TimeSeriesApi Response or something -- would be the same except without the query field